android/vision/app/src/main/res/drawable/media_pipe_banner.xml

```diff
@@ -0,0 +1,69 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Need to replace media banner with LabGraph vision banner.

Reply:
Resolved this!